src/test/java/com/google/api/codegen/discovery/DocumentTest.java

```diff
@@ -0,0 +1,67 @@
+/* Copyright 2016 Google Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.api.codegen.discovery;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.truth.Truth;
+import java.io.*;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import org.junit.Test;
+
+public class DocumentTest {
+  @Test
+  public void testDocument() throws IOException {
+    String file = "src/test/java/com/google/api/codegen/discoverytestdata/document.json";
+    Reader reader = new InputStreamReader(new FileInputStream(new File(file)));
+
+    ObjectMapper mapper = new ObjectMapper();
+    JsonNode root = mapper.readTree(reader);
+
+    Document document = Document.from(new DiscoveryNode(root));
+    Truth.assertThat(document.description().equals("My API!"));
```

Comment:
Two errors here:

1. You aren't calling a function on the result of `.assertThat()`
2. When you do that, you need `isEqualsTo`, not `equals`

It appears that this is the case for all of your assertions - you aren't testing anything.